pkg/api/constants.go

```diff
@@ -1,6 +1,46 @@
 package api
 
+import "fmt"
+
 const (
-	APIEndpoint     = "https://api.opensauced.pizza/v1"
-	BetaAPIEndpoint = "https://beta.api.opensauced.pizza/v1"
+	LocalhostEndpointKey = iota
+	ProdEndpointKey
+	BetaEndpointKey
+	AlphaEndpointKey
+	InvalidEndpointKey   = 666
+	LocalhostEndpointURL = "http://localhost:3001"
+	ProdEndpointURL      = "https://api.opensauced.pizza"
+	BetaEndpointURL      = "https://beta.api.opensauced.pizza"
+	AlphaEndpointURL     = "https://alpha.api.opensauced.pizza"
+	ToolsEndpointURL     = "https://opensauced.tools"
 )
+
+func EndpointKeyToURL(endpointKey int) (string, error) {
+	switch endpointKey {
+	case LocalhostEndpointKey:
+		return LocalhostEndpointURL, nil
+	case ProdEndpointKey:
+		return ProdEndpointURL, nil
+	case BetaEndpointKey:
+		return BetaEndpointURL, nil
+	case AlphaEndpointKey:
+		return AlphaEndpointURL, nil
+	default:
+		return "", fmt.Errorf("unknown endpoint key %d", endpointKey)
+	}
+}
```

Comment:
I'm generally not a fan of this approach. It always seems redundant. Why not just have callers get the endpoints directly through imports?

```go
myEndpoint := api.BetaEndpointURL
```

and I'm abit confused why we would need this? Generally, if a user provides `--endpoint`, we should take that whole sale and not check it against our keying of endpoints hard coded into the CLI. I may deploy my own API behind a local network and I want to provide that explicitly like:

```
pizza --endpoint https://api.opensauced.my-local-network.homelab ...
```

Reply:
For some reason, I assumed the go api client can only be used with the server configurations that it originally has.
I've refactored it to allow the use of any endpoint url.